examples/docs/zh-CN/i18n.md

```diff
@@ -68,6 +68,22 @@ Vue.locale('zh-cn', zhLocale)
 Vue.locale('en', enLocale)
 ```
 
+## 兼容其他 i18n 插件
+如果不使用 `vue-i18n`，而是用其他的 i18n 插件，Element 将无法兼容，但是可以自定义 Element 的 i18n 的处理方法。
```

Comment:
Element may not be compatible with i18n plugins other than `vue-i18n`, but you can customize how Element processes i18n.